tf_delete_client:
  delete_tf_frame: move_tf
  tf_delete_service_name: tf_delete_service